src/mixins/controls.js

```diff
@@ -1,5 +1,9 @@
 import * as THREE from "three";
+import {CSS2DRenderer} from "three-css2drender"
```

Comment:
Let's try and import CSS2DRenderer from three.js directly, and update the three.js version, if necessary